x-pack/docs/en/security/securing-communications/security-basic-setup-https.asciidoc

```diff
@@ -0,0 +1,595 @@
+
+. On each node, edit the `elasticsearch.yml` file to enable HTTPS security and
+   specify the location of the `http.p12` security certificate.
++
+[source,yaml]
+----
+xpack.security.http.ssl.enabled: true
+xpack.security.http.ssl.keystore.path: http.p12
+----
+
+. Add the password for your private key to a secure setting in {es}.
++
+[source,shell]
+----
+./bin/elasticsearch-keystore add xpack.security.http.ssl.keystore.secure_password
+----
+
+. Restart {es}.
+
+**Next**: <<encrypt-kibana-http,Encrypt HTTP client communications for {kib}>>
+
+[[encrypt-kibana-http]]
+==== Encrypt HTTP client communications for {kib}
+
+Browsers send traffic to {kib} and {kib} sends traffic to {es}.
+These communication channels are configured separately to use TLS. You encrypt
+traffic between your browser and {kib}, and then encrypt traffic between
+{kib} and {es}.
+
+[[encrypt-kibana-browser]]
+===== Encrypt traffic between your browser and {kib}
+
+You create a server certificate and private key for {kib}. {kib} uses this
+server certificate and corresponding private key when receiving connections
+from web browsers.
+
+When you obtain a server certificate, you must set its subject alternative
+name (SAN) correctly to ensure that browsers with hostname verification will
+trust it. You can set one or more SANs to the {kib} server’s fully-qualified
+domain name (FQDN), hostname, or IP address. When choosing the SAN, pick
+whichever attribute you'll use to connect to {kib} in your browser, which is
+likely the FQDN.
+
+The following instructions create an untrusted browser certificate for {kib}.
+This certificate is acceptable for development environments and building a
+proof of concept, but should not be used in a production environment.
+
+WARNING: Before going to production, use a trusted CA such as https://letsencrypt.org/[Let's
+Encrypt] or your organization's internal CA to sign the certificate. Using a
+signed certificate establishes browser trust for connections to {kib} for
+internal access or on the public internet.
+
+. Generate a server certificate and private key for {kib}.
++
+[source,shell]
+----
+./bin/elasticsearch-certutil csr -name kibana-server -dns example.com,www.example.com
+----
++
+The CSR has a common name (CN) of `kibana-server`, a SAN of `example.com`,
+and another SAN of `www.example.com`.
++
+This command generates a `csr-bundle.zip` file by default with the following
+contents:
++
+[source,txt]
+----
+/kibana-server
+|_ kibana-server.csr
+|_ kibana-server.key
+----
+
+. Unzip the `csr-bundle.zip` file to obtain the `kibana-server.csr` unsigned
+security certificate and the `kibana-server.key` unencrypted private key.
+
+. Send the `kibana-server.csr` unsigned security certificate to your internal
+CA or trusted CA for signing. The signed file should be a `.crt` file, such as
+`kibana-server.crt`.
++
+WARNING: {kib} will not trust the CSR. You must send the `kibana-server.csr`
```

Comment:
CSR is not a certificate. So technically Kibana just cannot use it as one, it is not about trust or not.

Reply:
Thanks for the clarification! I can change to read as:

> Kibana cannot use the CSR. You must send the `kibana-server.csr` unsigned security certificate to your internal CA or trusted CA for signing to obtained a signed certificate.